src/socket.c

```diff
@@ -2049,6 +2049,11 @@ struct sock *camblet_accept(struct sock *sk, int flags, int *err, bool kern)
 		client_sk->sk_user_data = sc;
 		WRITE_ONCE(client_sk->sk_prot, prot);
 	}
+	else
```

Comment:
If something happens in line 2011, the opa_socket_ctx and conn_ctx will be left behind since sc will only be set in line 2017, thus the error part cannot free them.

Reply:
Thanks , fixed!